ccp/modules/login-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ "$ENABLE_LOGIN" == true ];then
+  log INFO "Login setup detected -- will start Login services."
+  OVERRIDE+=" -f ./$PROJECT/modules/login-compose.yml"
+fi
```

Comment:
Where is this variable set?

Reply:
This should be defined in the ccp.conf file